subsys/bluetooth/audio/cap.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <bluetooth/gatt.h>
+
+#if defined(CONFIG_BT_CAP_ACCEPTOR)
```

Comment:
Should this be CONFIG_BT_CAS?  If not - what is the purpose of the CONFIG_BT_CAS?
(Maybe the answer will become clear in the next commit.)

Reply:
Removed `CONFIG_BT_CAS` as it held no value